components/dialog/dialog-mixin.js

```diff
@@ -14,7 +14,8 @@ import { tryGetIfrauBackdropService } from '../../helpers/ifrauBackdropService.j
 window.D2L = window.D2L || {};
 window.D2L.DialogMixin = window.D2L.DialogMixin || {};
 
-window.D2L.DialogMixin.hasNative = (window.HTMLDialogElement !== undefined);
+window.D2L.DialogMixin.hasNative = (window.HTMLDialogElement !== undefined)
+	&& (navigator.vendor && navigator.vendor.toLowerCase().indexOf('apple') === -1);
```

Comment:
I tested this and the vendor is still "Apple Computer Inc." even in Chrome/Firefox on iOS, which is what we want.